app/utils/sanity/type.go

```diff
@@ -79,7 +94,7 @@ type conflictPair struct {
 
 // checkConflict: check configA vs configB
 // and the value is equal to the conflicts value then complain it
-func (cp *conflictPair) checkConflict() error {
+func (cp *conflictPair) Check() error {
```

Comment:
Maybe use `check` is enough, we do not need export this function? same with `dependentPair`'s `Check`
```suggestion
func (cp *conflictPair) check() error {
```